charts/prometheus-adapter/templates/deployment.yaml

```diff
@@ -82,6 +82,10 @@ spec:
         readinessProbe:
           {{- toYaml . | nindent 10 }}
         {{- end }}
+        {{- with .Values.startupProbe }}
```

Comment:
Need to add a default `startupProbe: []` with a descriptive comment to the values.yaml